Bottom: f5b8fb13db6f4413bd1b65df6efcbf6361b0bc95
Top:    09a8ca0766abe169ca8cd2923887e07fbb83a3a8
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-19 20:32:16 +0800

Refresh of vhost_net-introduce-tx

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index e018940..a014ca0 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -498,7 +498,8 @@ static int get_tx_bufs(struct vhost_net *net,
 		return ret;
 
 	if (*in) {
-		vq_err(vq, "Unexpected descriptor format for TX: out %d, int %d\n", *out, *in);
+		vq_err(vq, "Unexpected descriptor format for TX: out %d, int %d\n",
+			*out, *in);
 		return -EFAULT;
 	}
